### ISSUE TITLE
Pulumi program for deploying to AWS

### ISSUE THREAD

**Issue by scottslowe** [siderolabs/contrib#16]
I've written a Pulumi program in Go to deploy Talos Linux to AWS. Is this something you'd be interested in a PR for?

**Comment by frezbo** [comment]
Yes, that'd be awesome. Do note that we'll be doing a new v0.2.0 of the TF provider (which eventually means a new pulumi provider release) with some major changes, so probably needs some code change if you don't mind. Else create a PR and we'll cleanup later.

**Comment by scottslowe** [comment]
@frezbo Cool, thanks. No worries on the provider release; I can update the code later (but glad to see that the provider is getting some attention). I noted that you (and others) seem to have a very specific code structure and formatting preferences for your Pulumi examples; are you strongly wedded to those?

**Comment by dirien** [comment]
The Pulumi examples are very outdated. I created a DigitalOcean example as I needed to recreate a question of a community member.

@frezbo is there any preference for the language the Pulumi example should be written in? I created the DO example in TS but saw that the existing examples are in Golang
